phpcs-ruleset.xml.dist

```diff
@@ -26,7 +26,6 @@
         <exclude name="DrupalPractice.Constants" />
         <exclude name="DrupalPractice.FunctionDefinitions" />
         <exclude name="DrupalPractice.General" />
-        <exclude name="DrupalPractice.InfoFiles" />
```

Comment:
Why are we removing this rule?

Reply:
It is being removed from the _exclude list_, meaning that it is in fact being added!